src/frequenz/client/dispatch/_client.py

```diff
@@ -166,7 +172,9 @@ async def create(
             recurrence=recurrence or RecurrenceRule(),
         )
 
-        await self._stub.CreateMicrogridDispatch(request.to_protobuf())  # type: ignore
+        await self._stub.CreateMicrogridDispatch(
+            request.to_protobuf(), metadata=self._metadata
+        )  # type: ignore
```

Comment:
The `cast` comment applied to all `type: ignore`, I would also use a `cast(Awaitable[ResponseType], ...)` here and in all stub calls.

Reply:
That seems to be out of the scope of this PR though

Reply:
True.

Reply:
https://github.com/frequenz-floss/frequenz-client-dispatch-python/issues/57